components/uuf-renderablecreator-hbs/src/main/java/org/wso2/carbon/uuf/renderablecreator/hbs/helpers/registry/RuntimeHelperRegistry.java

```diff
@@ -65,9 +65,9 @@
  *
```

Comment:
Change the first sentence of this javadoc to `Handlebars helpers registry for runtime.`